kubernetes/linux/main.sh

```diff
@@ -38,6 +38,68 @@ waitforlisteneronTCPport() {
       fi
 }
 
+#Setting environment variable for CAdvisor metrics to use port 10255/10250 based on curl request
```

Comment:
This is necessary to move earlier in main.sh to know the container runtime to change the fluent-bit conf file before starting fluent-bit